pkg/runner/runner.go

```diff
@@ -56,6 +60,7 @@ var (
 	appVersion         = pgmodel.VersionInfo{Version: version.Version, CommitHash: version.CommitHash}
 	migrationLockError = fmt.Errorf("Could not acquire migration lock. Ensure there are no other connectors running and try again.")
 	startupError       = fmt.Errorf("startup error")
+	HAState            = &ha.HA{}
```

Comment:
should be moved in the Run function, can be a local var init'ed there and passed to the components that need it